test/suites/dev/moonbase/test-eth-tx/test-eth-tx-native-transfer.ts

```diff
@@ -0,0 +1,59 @@
+import { describeSuite, expect, beforeEach } from "@moonwall/cli";
+import {
+  ALITH_ADDRESS,
+  BALTATHAR_ADDRESS,
+  GLMR,
+  createViemTransaction,
+  checkBalance,
+} from "@moonwall/util";
+
+describeSuite({
+  id: "D010307",
+  title: "Native Token Transfer Test",
+  foundationMethods: "dev",
+  testCases: ({ context, it }) => {
+    let initialAlithBalance: bigint;
+    let initialBaltatharBalance: bigint;
+
+    beforeEach(async function () {
+      initialAlithBalance = await checkBalance(context, ALITH_ADDRESS);
+      initialBaltatharBalance = await checkBalance(context, BALTATHAR_ADDRESS);
+    });
+
+    it({
```

Comment:
Please add a second test that estimate Gas and expect the estimation to equal 21000

Reply:
Done